platform/mikro-e/contiki-conf.h

```diff
@@ -36,6 +36,7 @@
 #define DEBUG_HIGHER_LEVELS 0
 
 #include <inttypes.h>
+#include "ca8210-conf.h"
```

Comment:
this line was removed in https://github.com/CreatorDev/contiki/pull/36/commits/b5d599f60ef43fe1b6b444af6c69e762e84bd0ab commit, any reason for adding it back ?